src/cli/cli.cpp

```diff
@@ -2498,7 +2500,7 @@ otError Interpreter::ProcessPskc(uint8_t aArgsLength, char *aArgs[])
         const otPskc *pskc = otThreadGetPskc(mInstance);
 
         OutputBytes(pskc->m8);
-        OutputLine("");
+        OutputFormat("\r\n");
```

Comment:
What is the reason for replacing the `OutputLine("")` with `OutputFormat("\r\n")`?
`OutputLine()` already should add the `\r\n`.